packages/blocks/src/__internal__/rich-text/rich-text-operations.ts

```diff
@@ -328,6 +328,9 @@ export function tryMatchSpaceHotkey(
   if (offset > prefix.length) {
     return ALLOW_DEFAULT;
   }
+  if (model.flavour === 'affine:code') {
```

Comment:
`matchFlavour` would be appreciated here

Reply:
Thanks! Applied